source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -23,14 +23,78 @@ By the end of the tutorial, you will have done the following:
 It is assumed you have the following:
 
  - You have [created an environment for your application][env-create]
- - You have [deployed an application][deploy-helm] to the 'cloud-platform-live-1' cluster using Helm.
+ - You have deployed the multi-container rails app to the 'cloud-platform-live-1' cluster.
```

Comment:
Can you make "multi-container rails app" a link to the user-guide article on deploying the app?

Reply:
done.